libs/wingsdk/src/shared-aws/bucket.inflight.ts

```diff
@@ -107,10 +119,18 @@ export class BucketClient implements IBucketClient {
    * @returns content of the object
    */
   public async tryGet(key: string): Promise<string | undefined> {
-    if (await this.exists(key)) {
-      return this.get(key);
+    const objectContent = await this.getObjectContent(key);
+    if (objectContent) {
```

Comment:
This can actually go into `getObjectContent` to avoid duplications, returning either a string or undefined- then in `tryGet` it will be returned, and in `get` an error will be thrown if undefined